### ISSUE TITLE
Failed to open file

### ISSUE THREAD

**Issue by ZereX666** [ClayAmore/ER-Save-Editor#72]
Failed to open file: Parse error: Failed to fetch regulation size from version: 11320031
![image](https://github.com/user-attachments/assets/f3391336-b415-407d-b919-6244a9c8bb35)

**Comment by IDStationX** [comment]
bro help me one my question

**Comment by brtrombin** [comment]
> Failed to open file: Parse error: Failed to fetch regulation size from version: 11320031 ![image](https://private-user-images.githubusercontent.com/65896858/356838594-f3391336-b415-407d-b919-6244a9c8bb35.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzMjc1NzgsIm5iZiI6MTcyMzMyNzI3OCwicGF0aCI6Ii82NTg5Njg1OC8zNTY4Mzg1OTQtZjMzOTEzMzYtYjQxNS00MDdkLWI5MTktNjI0NGE5YzhiYjM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEwVDIyMDExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2MjU4MjVkN2EzMTY2OTIxMjFjYzg4YjJkMDlmNmY0Zjc2YzUxMDkwNWRiZDRjZTU5YmMzOTY3NGI0Mjc5YWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AkxEoyMc-FA_euC7RQAc3UWHbV84ScqCv52v5k1YaD0)

I'm getting the same error, any idea? tried recompile and didn`t work

**Comment by ZereX666** [comment]
> > Failed to open file: Parse error: Failed to fetch regulation size from version: 11320031 ![image](https://private-user-images.githubusercontent.com/65896858/356838594-f3391336-b415-407d-b919-6244a9c8bb35.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzMjc1NzgsIm5iZiI6MTcyMzMyNzI3OCwicGF0aCI6Ii82NTg5Njg1OC8zNTY4Mzg1OTQtZjMzOTEzMzYtYjQxNS00MDdkLWI5MTktNjI0NGE5YzhiYjM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEwVDIyMDExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2MjU4MjVkN2EzMTY2OTIxMjFjYzg4YjJkMDlmNmY0Zjc2YzUxMDkwNWRiZDRjZTU5YmMzOTY3NGI0Mjc5YWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AkxEoyMc-FA_euC7RQAc3UWHbV84ScqCv52v5k1YaD0)
> 
> I'm getting the same error, any idea? tried recompile and didn`t work

possible game update, idk

**Comment by IDStationX** [comment]
when i put the ps4 save on ER editor the program closed

**Comment by ZereX666** [comment]
> > > Failed to open file: Parse error: Failed to fetch regulation size from version: 11320031 ![image](https://private-user-images.githubusercontent.com/65896858/356838594-f3391336-b415-407d-b919-6244a9c8bb35.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzMjc1NzgsIm5iZiI6MTcyMzMyNzI3OCwicGF0aCI6Ii82NTg5Njg1OC8zNTY4Mzg1OTQtZjMzOTEzMzYtYjQxNS00MDdkLWI5MTktNjI0NGE5YzhiYjM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEwVDIyMDExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2MjU4MjVkN2EzMTY2OTIxMjFjYzg4YjJkMDlmNmY0Zjc2YzUxMDkwNWRiZDRjZTU5YmMzOTY3NGI0Mjc5YWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AkxEoyMc-FA_euC7RQAc3UWHbV84ScqCv52v5k1YaD0)
> > 
> > 
> > I'm getting the same error, any idea? tried recompile and didn`t work
> 
> possible game update, idk

yes, old saves are opening

**Comment by IDStationX** [comment]
can u give me a one old save?

**Comment by ZereX666** [comment]
> can u give me a one old save?

i play from pc (steam). Here the save is linked to steamid

**Comment by IDStationX** [comment]
dont metter. i can put pc save convert to ps4

**Comment by ClayAmore** [comment]
Should work now! I updated the lib.

**Comment by iMakeMehPrograms** [comment]
Opening the test file provided with ER-Save-Lib causes the editor to freeze and stop responding until it loads whilst opening my organic save file causes this issue... Strange... (this is *actually* on 0.1.0 now that I've read up on my git). Also happens now that I've updated to latest commit on release/0.1.0.

**Comment by ClayAmore** [comment]
If youre running debug then its gonna take a long time before the file gets parsed. Try release build.

`cargo build --release` or `cargo run --release`

**Comment by iMakeMehPrograms** [comment]
I got it working! Turns out cargo wasn't pulling the latest commit, so I had to manually edit the cargo.lock to the commit with the fix. Thanks for the help! Unfortunately, I couldn't revive Radagon and get a new ending, as the golden wall's interact message didn't reappear no matter what settings I changed. Nor did Marika's interact reappear when going to the interior site of grace. Thanks for the help though!

**Comment by ClayAmore** [comment]
Yeah there's still stuff to do. Open an issue with the problem. Label as bug. I'll close this issue once that's done.
